Application/Frontend/pages/testingFile.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ChatMembersFinder from '../components/FriendsColumn/get_chat_members'; // Adjust the import path according to your project structure
+
+function TestPage() {
+  return (
+    <div>
+      <h1>Find members of chat group</h1>
```

Comment:
Again, please use components from the `Mantine` UI library (e.g. `<Text>`).

Reply:
test is just a test file it will be removed